airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/InlineEnrollmentCallout.module.scss

```diff
@@ -6,6 +6,10 @@
   width: 100%;
 }
 
+.withBottomMargin {
+  margin-bottom: variables.$spacing-lg;
+}
```

Comment:
The alignment here is odd.  The margin is the same top and bottom, but it's off-center thanks to some weirdness in other components.
 
<img width="866" alt="Screenshot 2023-02-15 at 1 20 41 PM" src="https://user-images.githubusercontent.com/63751206/219118415-c05380a2-7975-49d2-9d71-0d2c6ee927ab.png">

<img width="901" alt="Screenshot 2023-02-15 at 1 22 53 PM" src="https://user-images.githubusercontent.com/63751206/219118749-1fbf8559-6cb6-4c35-809a-d66c82d9d1dd.png">